### DISCUSSION TITLE
hi,about IModelTileRpcInterface.getClient().requestElementGraphics

### DISCUSSION THREAD

**Discussion by shenshengyi** [iTwin/itwinjs-core#972] [Q&A] [answered]
hi.
I'm using IModelTileRpcInterface.getClient().requestElementGraphics, This function returns the uint8array type. How can I display it through the decorator?

Again, only the graphic data of graphic elements can be obtained in this way? Or can we also get solid graphics data? Thanks.

**Comment by pmconne** [comment, marked as answer]
Please consult [this sample](https://www.itwinjs.org/sample-showcase/?group=Viewer+Features&sample=explode-sample&imodel=House+Sample).
Please explain what distinction you are trying to make between "graphic data of graphic elements" and "solid graphics data".

**Reply by shenshengyi** [reply]
very thanks.
